GildedRoseKata/BackstagePassAlter.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace GildedRoseKata
+{
+    internal class BackstagePassAlter : ItemAlter
+    {
+        public override void UpdateItemQuality(Item item)
+        {
+            item.SellIn--;
+
+            if (item.SellIn < 0)
+            {
+                item.Quality = item.Quality - item.Quality;
+            }
+            else if (item.SellIn < 5)
+            {
+                item.Quality += 3;
+            }
+            else if (item.SellIn < 10)
+            {
+                item.Quality += 2;
+            }
+            else
+            {
+                item.Quality += 1;
+            }
+
```

Comment:
Ugly if /else if blocks.

This is c# core, we have pattern matching 

We can write something like 

```suggestion
item.Quality = item.SellIn switch
{
    < 0 : 0,
    < 5 : item.Quality + 3 
    ...
};

```